packages/orchestrator-ui-components/src/components/WfoStartButton/WfoStartWorkflowComboBox.tsx

```diff
@@ -28,6 +28,7 @@ export const WfoStartWorkflowButtonComboBox = () => {
                 productId: option.productId,
             },
             label: option.productName || '',
+            disabled: !option.isAllowed,
```

Comment:
This makes it so it defaults to disabled correct? I would prefer it to 
```suggestion
            disabled: option.isAllowed && option.isAllowed === false,
```